src/main/java/frc/team2412/robot/commands/intake/IntakeExtendCommand.java

```diff
@@ -8,7 +8,6 @@
 
     public IntakeExtendCommand(IntakeSubsystem subsystem) {
         this.subsystem = subsystem;
-        addRequirements(subsystem);
```

Comment:
Sorry if I wasn't more clear in my last comment - classes like `IntakeExtendCommand`, which are not command groups, _do_ need to call `addRequirements()`, while classes that extend command groups like `IntakeInCommand` do not.